yoticlient.go

```diff
@@ -99,23 +94,25 @@ func (client *Client) GetSdkID() string {
 
 // GetActivityDetails requests information about a Yoti user using the one time use token generated by the Yoti login process.
 // It returns the outcome of the request. If the request was successful it will include the users details, otherwise
-// it will specify a reason the request failed.
-func (client *Client) GetActivityDetails(token string) (ActivityDetails, []string) {
-	_, activity, errStrings := client.getActivityDetails(token)
-	return activity, errStrings
+// an error it will be returned which will specify the reason the request failed,
```

Comment:
```suggestion
// an error will be returned, which will specify the reason the request failed.
```
On top of this, it looks like the two lines above are longer, and this one and the one below are shorter. 
Don't mind which one we go for (slight preference on shorter lines), but can we make them all the same 👍